chia/cmds/farm_funcs.py

```diff
@@ -183,11 +185,21 @@ async def challenges(farmer_rpc_port: int, limit: int) -> None:
 
 
 async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int, farmer_rpc_port: int) -> None:
-    amounts = await get_wallets_stats(wallet_rpc_port)
     plots = await get_plots(harvester_rpc_port)
     blockchain_state = await get_blockchain_state(rpc_port)
     farmer_running = await is_farmer_running(farmer_rpc_port)
 
+    bWalletNotReady = False
+    bWalletNotRunning = False
```

Comment:
I don't think we use hungarian notation in our codebases. I think it would be better to add type annotations for mypy instead

Reply:
Agreed - fixed